pages/blog/websocket-part3.md

```diff
@@ -574,8 +574,6 @@ Therefore, all descriptions and even things like **operationId** should have two
 
 > Join the discussion around [operationId](https://github.com/asyncapi/spec/issues/538).
 
-In my opinion, this limitation should not block you from choosing the direction of API-first. The benefits are too big to resign that easily. Just join us and let us find solutions together. We want to help solve all those issues, but we just need some help from you too.
```

Comment:
This sentence is almost word-for-word a copy of the sentence in the conclusion paragraph

Reply:
you are right, thanks